_layouts/downloadpage.html

```diff
@@ -16,60 +16,65 @@
             </div>
           </div>
           <div class="step">
             <div class="number-step">2</div>
             <div class="text-step">
-              <h3>Then, install Scala:</h3>
-              <p>...either by installing an IDE such as IntelliJ, or sbt, Scala's build tool.</p>
+              <h3><i>(Optional)</i> Then, install an IDE for Scala.</h3>
+              <p>The two most popular IDEs for Scala are IntelliJ and VS Code with Metals.</p>
             </div>
           </div>
           <div class="download-options">
             <div class="download-intellij">
               <span class="or">or</span>
               <div class="description">
                 <img src="/resources/img/download/arrow-left.png" alt="">
-                <p>Best if you prefer a full-featured IDE (recommended for beginners)</p>
+                <p>Best if you prefer to start without a build tool (recommended for beginners)</p>
               </div>
               <a href="" class="btn-download" id="download-intellij-link">
                 <i class="fa fa-download"></i>
-                <span>Download intellij</span>
+                <span>Download IntelliJ</span>
               </a>
               <ul>
-                {% include tutorial-list.html column=1 %}
+                <li><a href="/documentation/getting-started-intellij-track/getting-started-with-scala-in-intellij.html"><i class="fa fa-file-text-o"></i>Getting Started with Scala in IntelliJ</a></li>
+                <li><a href="/documentation/getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.html"><i class="fa fa-file-text-o"></i>Building a Scala Project with IntelliJ and sbt</a></li>
+                <li><a href="/documentation/getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.html"><i class="fa fa-file-text-o"></i>Testing Scala in IntelliJ with ScalaTest</a></li>
               </ul>
             </div>
-            <div class="download-sbt">
+            <div class="download-vscode">
               <div class="description">
                 <img src="/resources/img/download/arrow-right.png" alt="">
-                <p>Best if you are familiar with the command line</p>
+                <p>Best if you are familiar with the command line and build tools</p>
               </div>
-              <a href="" class="btn-download" id="download-sbt-link">
+              <a href="" class="btn-download" id="download-vscode-link">
                 <i class="fa fa-download"></i>
-                <span>Download Sbt</span>
+                <span>Download VS Code</span>
               </a>
               <ul>
-                {% include tutorial-list.html column=0 %}
+                <li><a href="https://scalameta.org/metals/docs/editors/vscode.html"><i class="fa fa-file-text-o"></i>Getting Started with Scala in VS Code</a></li>
```

Comment:
I would not qualify this tutorial as “getting started in Scala” tutorial. It seems to target developers already familiar with Scala. I think eventually we need to create a proper tutorial, like the ones below.

Reply:
So maybe it’s simpler to leave this section as it used to be and do the changes that you suggest when we have a proper “getting started” tutorial with VS Code + Metals?

Reply:
Reverted to the old content with sbt.